### ISSUE TITLE
There is no suggestion to save scenario on close

### ISSUE THREAD

**Issue by barancev** [SeleniumHQ/selenium-ide#361]
## 🐛 Bug Report

Open IDE, recods something, stop recorder and close the IDE window.

It's expected to be a dialog that suggest to save the project to prevent data loss. Actually, there is no one. 

Screencast: https://www.screencast.com/t/UN1m1R4y

## Environment

OS: Windows 7
Selenium IDE Version:  3.2.4
Browser:
Browser Version: Firefox 61

**Comment by corevo** [comment]
![save on exit](https://user-images.githubusercontent.com/888886/44515782-13c4bd80-a6cc-11e8-943a-f591065c9a90.gif)  
Maybe a windows issue? I'm on Firefox 61 as well.

**Comment by Genosite** [comment]
It works well for me.

Windows 10
Chrome 68
Selenium IDE v 3.2.0

**Comment by barancev** [comment]
@Genosite Yes, it works for me too in Chrome. The issue is Firefox-specific.
@corevo I'll try MacOS and comment later.

**Comment by barancev** [comment]
Yes, there is a warning on MacOS, seems to be a Windows-specific issue.
